python_apps/api_clients/api_clients/utils.py

```diff
@@ -91,14 +91,14 @@ def __init__(self, name, url, logger=None, api_key=None):
             self.logger = logger
         self.auth = KeyAuth(api_key)
 
-    def __call__(self, _post_data=None, params=None, **kwargs):
+    def __call__(self, data: dict = None, params: dict = None, **kwargs):
         final_url = self.url.params(**kwargs).url()
         self.logger.debug(final_url)
         try:
-            if _post_data:
+            if data is not None:
                 res = requests.post(
                     final_url,
-                    data=_post_data,
+                    json=data,
```

Comment:
@paddatrapper Ok, half of this PR is rubbish, I clearly didn't understood how the zend framework works. Any correction or confirmation would be really appreciated (cc @Robbt @hairmare), but here is what I understand:
The airtime_mvc api actions don't expect a JSON body, but a JSON encoded string inside a "data" POST form parameter.

In regards to #1172 and similar issues, I think the api_client doesn't get the post `data` anyway.

Reply:
Yuck... That sounds like a mess from Zend. We can probably add a optional parameter to the api_client call to map the `data` into a POST form parameter (defaulting to False)